src/main/scala/io/druid/indexer/spark/SparkDruidIndexer.scala

```diff
@@ -86,7 +86,16 @@ object SparkDruidIndexer {
       s => {
         val p = new Path(s)
         val fs = p.getFileSystem(sc.hadoopConfiguration)
-        fs.getFileStatus(p).getLen
+        if (fs.getFileStatus(p).isDirectory) {
+          var sum = 0l
+          val children = fs.listFiles(p, false)
```

Comment:
`fs.listFiles` returns a `RemoteIterator`, which isn't a subclass of `Iterator` and as such, there is no implicit converters towards Scala types. I could do something like `Iterator.continually(Try { children.next().getLen }).takeWhile(_.isSuccess).map(_.get).sum`, but I'm not sure it's better. 

Which one you'd prefer?